README.md

```diff
@@ -1,9 +1,8 @@
 <p align="center"><img src="docs/sources/assets/logo_and_name.png" alt="Grafana Agent logo"></p>
 
-Grafana Agent is a vendor-neutral, batteries-included telemetry collector with
-configuration inspired by [Terraform][]. It is designed to be flexible,
-performant, and compatible with multiple ecosystems such as Prometheus and
-OpenTelemetry.
+Grafana Agent is an OpenTelemetry Collector distribution with configuration
```

Comment:
```suggestion
Grafana Agent is Grafana's distribution of the OpenTelemetry Collector distribution with configuration
```

Reply:
After discussing offline, we decided to go with the original verbiage for now.